doc/Language/exceptions.pod6

```diff
@@ -385,11 +385,13 @@ printing a backtrace along with the message:
 
 =head1 Control exceptions
 
-Control exceptions are raised when throwing an Exception which does the
-L<X::Control|/type/X::Control> role (since Rakudo 2019.03). They are
+=begin para :versioned<down>
```

Comment:
You're just making up these values, right? Or is `up` and `down` intended to actually mean something?

Reply:
They do mean something, but probably not what you think they do.

The intent of "up" and "down" is to indicate where this version note belongs to: to previous text or to subsequent text (see an arrow in the right upper corner pointing up in the sample image).

In cases where a version note starts a section or ends it, it is kind of ambiguous where it really belongs, so this is a visual helper and nothing more.

As for Raku versions to put in there: yes, this is what I want to actually settle in this thread. :)

Reply:
That would be `above` and `below`, then... But then it's more confusing. Why don't you simply mark the paragraph that's versioned, instead of the versioning note?

Reply:
>But then it's more confusing

I'll try to explain in an example.

For example, we have this:

```
method foo is such and such.

[ Oh, by the way, in version 2020.02 you can do this / it behaves this ]

method bar

```

Visually / out of context it is not always obvious to the user if this `Oh, by the way` refers to `foo` or to `bar`. This tiny visual hint tries to help with that.

This is not a replacement for telling in a configurable way "Since Rakudo ...", it is just something simple to make this thing move and we could discuss.